packages/experiments/src/components/PersonaCoin/PersonaCoin.types.ts

```diff
@@ -1,12 +1,14 @@
-import { ImageLoadState } from 'office-ui-fabric-react';
-import { IComponent, IComponentStyles, IHTMLDivSlot, ISlotProp, IStyleableComponentProps } from '../../Foundation';
+import { ImageLoadState, IRefObject } from 'office-ui-fabric-react';
+import { IComponentStyles, IHTMLDivSlot, ISlotProp, IComponent, IStyleableComponentProps } from '../../Foundation';
 import { IPersonaPresenceSlot } from '../../utilities/factoryComponents.types';
 import { IPersonaCoinImageSlot } from './PersonaCoinImage/PersonaCoinImage.types';
 
-export type IPersonaCoinComponent = IComponent<IPersonaCoinProps, IPersonaCoinTokens, IPersonaCoinStyles>;
+export type IPersonaCoinComponent = IComponent<IPersonaCoinProps, IPersonaCoinTokens, IPersonaCoinStyles, IPersonaCoinViewProps>;
 
 export type IPersonaCoinSlot = ISlotProp<IPersonaCoinProps> | string;
 
+export type IPersonaCoinForSmallestSizeSlot = ISlotProp<{}>;
```

Comment:
Is this TBD or a TODO? I think this is basically an `any`.

Reply:
It's a slot without props, and should still allow JSX/RenderProp overrides? (Maybe this is not how that should be implemented :P)

Reply:
Ok I did a bit more testing. This is basically an 'any' in that it allows any type of object to be passed. I'm going to change `ISlotProp` to default to `never`, which will then only allow render functions and JSX and will allow you to simply do this:
```tsx
export type IPersonaCoinForSmallestSizeSlot = ISlotProp;
```

Or just use `ISlotProp` directly.

Until I can merge this change in, for now, you can do:
```tsx
export type IPersonaCoinForSmallestSizeSlot = ISlotProp<never>;
```

Reply:
Ah yes ofcourse this way it'll basically accept any object :) 

Seems like your last suggestion won't work.

![image](https://user-images.githubusercontent.com/2134901/51337304-03e8b400-1a87-11e9-8639-d4b9db587f1e.png)

Reply:
Ok, I tested it but the component was probably passing in props. I'll take another look and get back.